src/components/accordion.ts

```diff
@@ -0,0 +1,78 @@
+import {Component, View, NgClass, OnDestroy} from 'angular2/angular2';
+
+@Component({
+  selector: 'ngb-accordion, [ngb-accordion]',
+  properties: ['onlyOneOpen: closeOthers'],
+  host: {'class': 'panel-group'}
```

Comment:
I am seeing that too, perhaps I can remove it, no wrapper needed. That doesn't resolve it in the dropdown tho.